apps/rush-lib/src/logic/Git.ts

```diff
@@ -29,6 +29,7 @@ export class Git {
   private _gitInfo: gitInfo.GitRepoInfo | undefined;
 
   private _gitEmailResult: IResultOrError<string> | undefined = undefined;
+  private _gitHooksPath: IResultOrError<string> | undefined = undefined;
```

Comment:
I've monkeyed the data structure here from `_gitEmailResult` above, but I'm not actually doing anything with any errors. I wonder if I should use something simpler, or this is fine?

Reply:
You could probably handle the error in a method similar to how we handle the error in `getGitEmail()` here: https://github.com/microsoft/rushstack/blob/0f2d6a586a16a6970f7d6614d652b3bdd5b086bf/apps/rush-lib/src/logic/Git.ts#L100